utils/firestore.js

```diff
@@ -2180,40 +2234,148 @@ const getParticipantSelection = async (filter) => {
     }
 }
 
-const assignKitToParticipants = async (data) => {
+const assignKitToParticipant = async (data) => {
     try {
-        const snapshot = await db.collection("kitAssembly").where('supplyKitId', '==', data.supply_kitId).where('supplyKitIdUtilized', '==', false).get();
-        if (Object.keys(snapshot.docs).length !== 0) {
-            snapshot.docs.map(doc => {
-                data['collection_cardId'] = doc.data().collectionCardId
-                data['collection_cupId'] = doc.data().collectionCupId
-                data['specimen_kitId'] = doc.data().specimenKitId
-                data['specimen_kit_usps_trackingNum'] = doc.data().uspsTrackingNumber
-            })
+        const snapshot = await db.collection("kitAssembly").where('690210658', '==', data['690210658']).where('221592017', '==', '517216441').get();
+        if(snapshot.size > 0) {
```

Comment:
Are we looking for multiple results here? Based on the name, it looks like a single participant. Maybe `=== 1` would be more explicit. Then, if it's ever !== 1, we'll know about the error.